src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowCollection.cs

```diff
@@ -8,6 +9,7 @@ public class ShadowCollection : ObservableCollection<Shadow>
 	public bool HasInnerShadow() => this.Any(s => s.IsInner);
 
 	public string ToKey(double width, double height, Windows.UI.Color? contentBackground)
-		=> $"w{width},h{height}" + (contentBackground.HasValue ? $",cb{contentBackground.Value}:" : ":") +
+		=> $"w{width.ToString(CultureInfo.InvariantCulture)},h{height.ToString(CultureInfo.InvariantCulture)}" +
```

Comment:
```suggestion
		=> FormattableString.Invariant($"w{width},h{height}") +
```

Reply:
In fact `string.Create(CultureInfo.InvariantCulture, ...)` is better than `FormattableString.Invariant` to prevent boxing: https://github.com/dotnet/runtime/issues/73639